examples/demo/site/themes/bootstrap/assets/js/roc_auth.js

```diff
@@ -0,0 +1,307 @@
+var ROC_AUTH = ROC_AUTH || { };
```

Comment:
in the future, we will need to move that to the related module.
And include it when needed.